LiveStream/Service/KsLive.swift

```diff
@@ -0,0 +1,39 @@
+import FirebaseAnalytics
+import FirebaseDatabase
+import FirebaseAuth
+
+public class KsLiveApp {
```

Comment:
Oh so that's an interesting observation, I initially tried to do that when `KsLive` was a separate framework, however this caused ambiguity when I wanted to call `KsLive.something()` from the main app so I had to rename the class to something that was not the same name as the framework. @mbrandonw has since refactored things so that this class doesn't exist at all but just thought I'd mention why it was there in the first place 😄